### PR TITLE
Hero for the homepage

### PR DESCRIPTION
**Pull request by LouisKamp**
This pull implements a Hero for the homepage. 
This Hero is an adaptation from [this](https://getbootstrap.com/docs/5.0/examples/heroes/).

Screenshot 2024-11-12 at 13 15 04

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,21 +1,12 @@
+:html_theme.sidebar_secondary.remove: true
 .. _homepage-reference:
 
 .. meta::
    :description: Technical University of Denmark (DTU) Python Installation Support
    :keywords: dtudk, dtu, support, python, python3, pip, pip3, conda, venv, virtualenv
 
-.. todolist::
```
Comment (zerothi) [inline docs/index.rst:7]: please undelete this, it should stay here for dev purposes.

File: docs/index.rst [hunk 2]
```diff
@@ -101,9 +92,7 @@ Office hours
    :hidden:
 
    install/python.rst
-   learn-more/vscode/index.rst
```
Comment (zerothi) [inline docs/index.rst:104]: why did you delete this entry?

Comment (LouisKamp) [reply docs/index.rst:104]: My bad, that is an artifact from an experiment that wasn't supposed to be included in this pull.

**Review by zerothi — APPROVED** [review]
